<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="pl" sourcelanguage="en">
  <context>
    <name>AddonsInstaller</name>
    <message>
      <location filename="install_to_toolbar.py" line="76"/>
      <source>Create new toolbar</source>
      <translation>Utwórz nowy pasek narzędzi</translation>
    </message>
    <message>
      <location filename="install_to_toolbar.py" line="219"/>
      <source>A macro installed with the FreeCAD Addon Manager</source>
      <translation>Makro zainstalowane przy pomocy Menedżera dodatków FreeCAD</translation>
    </message>
    <message>
      <location filename="install_to_toolbar.py" line="221"/>
      <source>Run</source>
      <comment>Indicates a macro that can be &apos;run&apos;</comment>
      <translation>Uruchom</translation>
    </message>
    <message>
      <location filename="package_details.py" line="112"/>
      <source>Version {version} installed on {date}</source>
      <translation>Wersja {version} została zainstalowana: {date}</translation>
    </message>
    <message>
      <location filename="package_details.py" line="117"/>
      <source>Version {version} installed</source>
      <translation>Zainstalowano wersję {version}</translation>
    </message>
    <message>
      <location filename="package_details.py" line="121"/>
      <source>Installed on {date}</source>
      <translation>Data instalacji {date}</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="567"/>
      <source>Installed</source>
      <translation>Zainstalowano</translation>
    </message>
    <message>
      <location filename="package_details.py" line="96"/>
      <source>Version</source>
      <translation>Wersja</translation>
    </message>
    <message>
      <location filename="package_details.py" line="103"/>
      <source>Unknown version (no package.xml file found)</source>
      <translation>Nieznana wersja (nie znaleziono pliku package.xml)</translation>
    </message>
    <message>
      <location filename="package_details.py" line="108"/>
      <source>installed on</source>
      <translation>zainstalowane w</translation>
    </message>
    <message>
      <location filename="package_details.py" line="118"/>
      <source>installed</source>
      <translation>zainstalowano</translation>
    </message>
    <message>
      <location filename="package_details.py" line="126"/>
      <source>Update available to version</source>
      <translation>Aktualizacja dostępna do wersji</translation>
    </message>
    <message>
      <location filename="package_details.py" line="144"/>
      <source>This is the latest version available</source>
      <translation>To jest najnowsza dostępna wersja</translation>
    </message>
    <message>
      <location filename="package_details.py" line="151"/>
      <source>Updated, please restart FreeCAD to use</source>
      <translation>Zaktualizowano, uruchom ponownie program FreeCAD aby używać</translation>
    </message>
    <message>
      <location filename="package_details.py" line="159"/>
      <source>Update check in progress</source>
      <translation>Sprawdzanie aktualizacji w toku</translation>
    </message>
    <message>
      <location filename="package_details.py" line="163"/>
      <source>Automatic update checks disabled</source>
      <translation>Automatyczne sprawdzanie aktualizacji wyłączone</translation>
    </message>
    <message>
      <location filename="package_details.py" line="172"/>
      <source>Installation location</source>
      <translation>Miejsce instalacji</translation>
    </message>
    <message>
      <location filename="package_details.py" line="240"/>
      <source>WARNING: This addon is obsolete</source>
      <translation>UWAGA: Ten dodatek jest przestarzały</translation>
    </message>
    <message>
      <location filename="package_details.py" line="250"/>
      <source>WARNING: This addon is Python 2 Only</source>
      <translation>UWAGA: Ten dodatek jest przeznaczony tylko dla środowiska Python 2</translation>
    </message>
    <message>
      <location filename="package_details.py" line="312"/>
      <source>Fetching README.md from package repository</source>
      <translation>Pobieranie README.md z repozytorium pakietów</translation>
    </message>
    <message>
      <location filename="package_details.py" line="354"/>
      <source>WARNING: This addon requires FreeCAD </source>
      <translation>OSTRZEŻENIE: Ten dodatek wymaga programu FreeCAD </translation>
    </message>
    <message numerus="yes">
      <location filename="package_details.py" line="431"/>
      <source>Delete cached version of this README and re-download</source>
      <translation>
        <numerusform>Usuń buforowaną wersję tego pliku README, i pobierz ponownie</numerusform>
        <numerusform>Usuń buforowaną wersję tych plików README, i pobierz ponownie</numerusform>
        <numerusform>Usuń buforowaną wersję tych plików README, i pobierz ponownie</numerusform>
        <numerusform>Usuń buforowaną wersję tych plików README, i pobierz ponownie</numerusform>
      </translation>
    </message>
    <message>
      <location filename="package_details.py" line="510"/>
      <source>No URL or wiki page provided by this macro</source>
      <translation>Brak adresu URL lub strony Wiki udostępnionej przez to makro</translation>
    </message>
    <message>
      <location filename="package_details.py" line="563"/>
      <source>Could not load README data from URL {}</source>
      <translation>Nie można wczytać pliku danych README z adresu URL {}</translation>
    </message>
    <message>
      <location filename="package_details.py" line="577"/>
      <source>Success</source>
      <translation>Zakończono pomyślnie</translation>
    </message>
    <message>
      <location filename="package_details.py" line="600"/>
      <source>Changed to git ref &apos;{}&apos; -- please restart to use Addon.</source>
      <translation>Zmieniono w Git Reference's &apos;{}&apos; - Ponowne uruchomienie jest wymagane do użycia dodatku.</translation>
    </message>
    <message>
      <location filename="package_details.py" line="636"/>
      <source>Page JavaScript reported</source>
      <translation>Raportowany JavaScript strony</translation>
    </message>
    <message>
      <location filename="package_details.py" line="407"/>
      <source>Install</source>
      <translation>Zainstaluj</translation>
    </message>
    <message>
      <location filename="package_details.py" line="410"/>
      <source>Uninstall</source>
      <translation>Odinstaluj</translation>
    </message>
    <message>
      <location filename="package_details.py" line="413"/>
      <source>Update</source>
      <translation>Aktualizuj</translation>
    </message>
    <message>
      <location filename="package_details.py" line="416"/>
      <source>Check for Update</source>
      <translation>Sprawdź aktualizacje</translation>
    </message>
    <message>
      <location filename="package_details.py" line="419"/>
      <source>Run Macro</source>
      <translation>Uruchom makrodefinicję</translation>
    </message>
    <message>
      <location filename="package_details.py" line="786"/>
      <source>Change Branch</source>
      <translation>Zmień gałąź</translation>
    </message>
    <message>
      <location filename="package_details.py" line="867"/>
      <source>Enable</source>
      <translation>Włącz</translation>
    </message>
    <message>
      <location filename="package_details.py" line="870"/>
      <source>Disable</source>
      <translation>Wyłącz</translation>
    </message>
    <message>
      <location filename="package_details.py" line="424"/>
      <source>Return to package list</source>
      <translation>Wróć do listy pakietów</translation>
    </message>
    <message numerus="yes">
      <location filename="package_details.py" line="800"/>
      <source>QtWebEngine Python bindings not installed -- using fallback README display. See Report View for details and installation instructions.</source>
      <translation>
        <numerusform>Nie zainstalowano powiązania QtWebEngine z Pythonem — użycie awaryjnego wyświetlania pliku README. Szczegółowe informacje i instrukcje instalacji można znaleźć w oknie Raportu widoku.</numerusform>
        <numerusform>Nie zainstalowano powiązań QtWebEngine z Pythonem — użycie awaryjnego wyświetlania pliku README. Szczegółowe informacje i instrukcje instalacji można znaleźć w oknie Raportu widoku.</numerusform>
        <numerusform>Nie zainstalowano powiązań QtWebEngine z Pythonem — użycie awaryjnego wyświetlania pliku README. Szczegółowe informacje i instrukcje instalacji można znaleźć w oknie Raportu widoku.</numerusform>
        <numerusform>Nie zainstalowano powiązań QtWebEngine z Pythonem — użycie awaryjnego wyświetlania pliku README. Szczegółowe informacje i instrukcje instalacji można znaleźć w oknie Raportu widoku.</numerusform>
      </translation>
    </message>
    <message>
      <location filename="package_list.py" line="119"/>
      <source>Filter is valid</source>
      <translation>Filtr jest prawidłowy</translation>
    </message>
    <message>
      <location filename="package_list.py" line="125"/>
      <source>Filter regular expression is invalid</source>
      <translation>Wyrażenie regularne filtra jest nieprawidłowe</translation>
    </message>
    <message>
      <location filename="package_list.py" line="207"/>
      <source>Click for details about package {}</source>
      <translation>Kliknij, aby uzyskać informacje o pakiecie {}</translation>
    </message>
    <message>
      <location filename="package_list.py" line="211"/>
      <source>Click for details about workbench {}</source>
      <translation>Kliknij, aby uzyskać informacje o środowisku pracy {}</translation>
    </message>
    <message>
      <location filename="package_list.py" line="215"/>
      <source>Click for details about macro {}</source>
      <translation>Kliknij, aby uzyskać informacje o makrodefinicji {}</translation>
    </message>
    <message>
      <location filename="package_list.py" line="326"/>
      <source>Maintainer</source>
      <translation>Opiekun</translation>
    </message>
    <message>
      <location filename="package_list.py" line="330"/>
      <source>Maintainers:</source>
      <translation>Opiekunowie:</translation>
    </message>
    <message>
      <location filename="package_list.py" line="364"/>
      <source>Tags</source>
      <translation>Znaczniki</translation>
    </message>
    <message>
      <location filename="package_list.py" line="371"/>
      <source>updated</source>
      <translation>zaktualizowano</translation>
    </message>
    <message>
      <location filename="package_list.py" line="382"/>
      <source>Author</source>
      <translation>Autor</translation>
    </message>
    <message>
      <location filename="package_list.py" line="404"/>
      <source>Up-to-date</source>
      <translation>Aktualny</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="586"/>
      <source>Update available</source>
      <translation>Dostępna aktualizacja</translation>
    </message>
    <message>
      <location filename="package_list.py" line="414"/>
      <source>Pending restart</source>
      <translation>Oczekuje na ponowne uruchomienie</translation>
    </message>
    <message>
      <location filename="package_list.py" line="498"/>
      <source>DISABLED</source>
      <translation>WYŁĄCZONY</translation>
    </message>
    <message>
      <location filename="package_list.py" line="372"/>
      <source>Installed version</source>
      <translation>Zainstalowana wersja</translation>
    </message>
    <message>
      <location filename="package_list.py" line="180"/>
      <source>Click for details about package</source>
      <translation>Kliknij, aby uzyskać informacje o pakiecie</translation>
    </message>
    <message>
      <location filename="package_list.py" line="185"/>
      <source>Click for details about workbench</source>
      <translation>Kliknij, aby uzyskać informacje o środowisku pracy</translation>
    </message>
    <message>
      <location filename="package_list.py" line="190"/>
      <source>Click for details about macro</source>
      <translation>Kliknij, aby uzyskać informacje o makrodefinicji</translation>
    </message>
    <message>
      <location filename="package_list.py" line="376"/>
      <source>Updated</source>
      <translation>Zaktualizowano</translation>
    </message>
    <message>
      <location filename="package_list.py" line="380"/>
      <source>Unknown version</source>
      <translation>Nieznana wersja</translation>
    </message>
    <message>
      <location filename="package_list.py" line="383"/>
      <source>Installed on</source>
      <translation>Zainstalowane w</translation>
    </message>
    <message>
      <location filename="package_list.py" line="394"/>
      <source>Available version</source>
      <translation>Dostępna wersja</translation>
    </message>
    <message>
      <location filename="package_list.py" line="566"/>
      <source>Show Addons containing:</source>
      <translation>Pokaż dodatki zawierające:</translation>
    </message>
    <message>
      <location filename="package_list.py" line="569"/>
      <source>All</source>
      <translation>Wszystkie</translation>
    </message>
    <message>
      <location filename="package_list.py" line="572"/>
      <source>Workbenches</source>
      <translation>Środowiska
    pracy</translation>
    </message>
    <message>
      <location filename="package_list.py" line="575"/>
      <source>Macros</source>
      <translation>Makrodefinicje</translation>
    </message>
    <message>
      <location filename="package_list.py" line="578"/>
      <source>Preference Packs</source>
      <translation>Pakiety preferencji</translation>
    </message>
    <message>
      <location filename="package_list.py" line="692"/>
      <source>Status:</source>
      <translation>Status:</translation>
    </message>
    <message>
      <location filename="package_list.py" line="695"/>
      <source>Any</source>
      <translation>Dowolny</translation>
    </message>
    <message>
      <location filename="package_list.py" line="702"/>
      <source>Not installed</source>
      <translation>Nie zainstalowano</translation>
    </message>
    <message>
      <location filename="package_list.py" line="582"/>
      <source>Filter</source>
      <translation>Filtr</translation>
    </message>
    <message>
      <location filename="package_list.py" line="585"/>
      <source>OK</source>
      <translation>OK</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="115"/>
      <source>Checking network connection...
</source>
      <translation>Sprawdzanie połączenia z siecią ...
</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="167"/>
      <source>Unable to download addon list.</source>
      <translation>Nie można pobrać listy dodatków.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="172"/>
      <source>Workbenches list was updated.</source>
      <translation>Lista Środowisk pracy została zaktualizowana.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="261"/>
      <source>Failed loading</source>
      <translation>Niepowodzenie ładowania</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="296"/>
      <source>List of macros successfully retrieved.</source>
      <translation>Pomyślnie pobrano listę makrodefinicji.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="310"/>
      <source>GitPython not installed! Cannot retrieve macros from git</source>
      <translation>GitPython nie jest zainstalowany! Nie można pobrać makrodefinicji z git</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="319"/>
      <source>Something went wrong with the Git Macro Retrieval, possibly the Git executable is not in the path</source>
      <translation>Coś poszło nie tak z pobieraniem makr Git, prawdopodobnie plik wykonywalny Git nie występuje w podanej lokalizacjii</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="343"/>
      <source>Appears to be an issue connecting to the Wiki, therefore cannot retrieve Wiki macro list at this time</source>
      <translation>Wygląda na to, że występuje problem z połączeniem się z Wiki, dlatego nie można obecnie pobrać listy makrodefinicji dostępnej na Wiki</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="367"/>
      <source>Unable to fetch git updates for workbench</source>
      <translation>Nie można pobrać aktualizacji git dla środowiska pracy</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="391"/>
      <source>Retrieving info from</source>
      <translation>Pobieranie informacji z</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="419"/>
      <source>Failed to read metadata from</source>
      <translation>Nie udało się odczytać metadanych z</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="426"/>
      <source>git fetch failed for {}</source>
      <translation>pobieranie z git nie powiodło się dla {}</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="429"/>
      <source>git fetch failed for {wb.name}</source>
      <translation>pobieranie z git nie powiodło się dla {wb.name}</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="433"/>
      <source>Raw markdown displayed</source>
      <translation>Wyświetlono surowy format markdown</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="435"/>
      <source>Python Markdown library is missing.</source>
      <translation>Brakuje biblioteki Markdown środowiska Python.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="439"/>
      <source>git pull failed for {wb.name}</source>
      <translation>pobieranie z git nie powiodło się dla {wb.name}</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="514"/>
      <source>Done locating macros.</source>
      <translation>Zakończono lokalizowanie makrodefinicji.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="521"/>
      <source>This addon is already installed.</source>
      <translation>Ten dodatek jest już zainstalowany.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="533"/>
      <source>An update is available for this addon.</source>
      <translation>Aktualizacja jest dostępna dla tego dodatku.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="547"/>
      <source>This addon is marked as obsolete</source>
      <translation>Ten dodatek jest oznaczony jako przestarzały</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="551"/>
      <source>This usually means it is no longer maintained, and some more advanced addon in this list provides the same functionality.</source>
      <translation>Oznacza to, że zazwyczaj nie jest on już utrzymywany, a niektóre bardziej zaawansowane dodatki z tej listy zapewniają tę samą funkcjonalność.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="559"/>
      <source>This addon is marked as Python 2 Only</source>
      <translation>Ten dodatek jest oznaczony tylko jako Python 2</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="564"/>
      <source>This workbench may no longer be maintained and installing it on a Python 3 system will more than likely result in errors at startup or while in use.</source>
      <translation>Te środowisko pracy nie może być dłużej utrzymywane, a zainstalowanie go w systemie Python 3 z dużym prawdopodobieństwem doprowadzi do błędów podczas uruchamiania lub podczas użytkowania.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="608"/>
      <source>An error occurred fetching macros from GitHub</source>
      <translation>Wystąpił błąd podczas pobierania makr z GitHub</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="646"/>
      <source>An error occurred updating macros from GitHub</source>
      <translation>Wystąpił błąd podczas aktualizacji makr z GitHub</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="651"/>
      <source>Retrieving description...</source>
      <translation>Pobieranie opisu...</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="665"/>
      <source>This macro is already installed.</source>
      <translation>Moduł jest już zainstalowany.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="697"/>
      <source>Caching macro code...</source>
      <translation>Pamięć podręczna makra...</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="700"/>
      <source>GitPython not found. Using standard download instead.</source>
      <translation>Nie znaleziono GitPython. Zamiast tego użyto standardowego pobierania.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="705"/>
      <source>Your version of python doesn&apos;t appear to support ZIP files. Unable to proceed.</source>
      <translation>Twoja wersja Pythona nie&apos; obsługuje plików ZIP. Nie można kontynuować.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="727"/>
      <source>User requested updating a Python 2 workbench on a system running Python 3 - </source>
      <translation>Użytkownik poprosił o zaktualizowanie Środowiska pracy Python 2 w systemie z obsługą Pythona 3 - </translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="738"/>
      <source>Outdated GitPython detected, consider upgrading with pip.</source>
      <translation>Wykryto przestarzały GitPython, rozważ aktualizację za pomocą programu pip.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="763"/>
      <source>Workbench successfully updated. Please restart FreeCAD to apply the changes.</source>
      <translation>Środowisko pracy zaktualizowano pomyślnie. Uruchom ponownie FreeCAD, aby zastosować zmiany.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="771"/>
      <source>User requested installing a Python 2 workbench on a system running Python 3 - </source>
      <translation>Użytkownik poprosił o zainstalowanie Środowiska pracy Python 2 w systemie z obsługą Pythona 3 - </translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="786"/>
      <source>Workbench successfully installed. Please restart FreeCAD to apply the changes.</source>
      <translation>Środowisko pracy zainstalowano pomyślnie. Uruchom ponownie FreeCAD, aby zastosować zmiany.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="799"/>
      <source>A macro has been installed and is available under Macro -&gt; Macros menu</source>
      <translation>Makro zostało zainstalowane i jest dostępne w menu Macrodefinicji -&gt; Menu makrodefinicji</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="824"/>
      <source>Retrieving info from {}</source>
      <translation>Pobieranie informacji z {}</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="835"/>
      <source>Missing workbench</source>
      <translation>Nieistniejące Środowisko pracy</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="844"/>
      <source>Missing python module</source>
      <translation>Nieistniejący moduł Pyton</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="854"/>
      <source>Missing optional python module (doesn&apos;t prevent installing)</source>
      <translation>Brakuje opcjonalnego modułu Python (nie przeszkadza w instalacji)</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="857"/>
      <source>Some errors were found that prevent to install this workbench</source>
      <translation>Znaleziono pewne błędy, które uniemożliwiają zainstalowanie tego środowiska</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="859"/>
      <source>Please install the missing components first.</source>
      <translation>Proszę najpierw zainstalować brakujące komponenty.</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="873"/>
      <source>Error: Unable to locate zip from</source>
      <translation>Błąd: nie można znaleźć pliku Zip z</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="880"/>
      <source>Error: Unable to download</source>
      <translation>Błąd: Nie można pobrać</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="893"/>
      <source>Successfully installed</source>
      <translation>Zainstalowano pomyślnie</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="915"/>
      <source>Retrieving macro description...</source>
      <translation>Pobieranie opisu makrodefinicji ...</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="653"/>
      <source>Retrieving info from git</source>
      <translation>Pobieranie informacji z git</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="656"/>
      <source>Retrieving info from wiki</source>
      <translation>Pobieranie informacji z Wiki</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1013"/>
      <source>Error updating module</source>
      <translation>Wystąpił błąd podczas aktualizacji modułu</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1075"/>
      <source>Error updating module </source>
      <translation>Błąd aktualizacji modułu </translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1078"/>
      <source>Please fix manually</source>
      <translation>Proszę naprawić ręcznie</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1220"/>
      <source>Error: Unable to locate ZIP from</source>
      <translation>Błąd: Nie można zlokalizować ZIP z</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1403"/>
      <source>Retrieving package metadata...</source>
      <translation>Pobieranie metadanych pakietu...</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1405"/>
      <source>Failed to execute pip. Returned error was:</source>
      <translation>Wykonanie "obrazu w obrazie" nie powiodło się. Zwrócony błąd:</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1611"/>
      <source>Downloaded package.xml for {}</source>
      <translation>Pobrano plik package.xml dla {}</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1641"/>
      <source>Downloaded metadata.txt for {}</source>
      <translation>Pobrano plik metadata.txt dla {}</translation>
    </message>
    <message>
      <location filename="addonmanager_workers.py" line="1715"/>
      <source>Downloaded icon for {}</source>
      <translation>Pobrano plik ikonki dla {}</translation>
    </message>
    <message>
      <location filename="addonmanager_utilities.py" line="332"/>
      <source>Failed to repair missing .git directory</source>
      <translation>Nie udało się naprawić brakującego katalogu .git</translation>
    </message>
    <message>
      <location filename="addonmanager_utilities.py" line="335"/>
      <source>Repository URL</source>
      <translation>Adres URL repozytorium</translation>
    </message>
    <message>
      <location filename="addonmanager_utilities.py" line="338"/>
      <source>Clone directory</source>
      <translation>Klonuj katalog</translation>
    </message>
    <message>
      <location filename="addonmanager_metadata.py" line="72"/>
      <source>Error with encrypted connection</source>
      <translation>Błąd z połączeniem szyfrowanym</translation>
    </message>
    <message>
      <location filename="addonmanager_metadata.py" line="135"/>
      <source>Failed to connect to</source>
      <translation>Nie udało się połączyć z</translation>
    </message>
    <message>
      <location filename="addonmanager_metadata.py" line="220"/>
      <source>Failed to connect to URL</source>
      <translation>Nie udało się połączyć z serwerem URL</translation>
    </message>
    <message>
      <location filename="change_branch.py" line="101"/>
      <source>DANGER: Developer feature</source>
      <translation>OSTRZEŻENIE: Funkcja dewelopera</translation>
    </message>
    <message>
      <location filename="change_branch.py" line="114"/>
      <source>There are local changes</source>
      <translation>Występują zmiany lokalne</translation>
    </message>
    <message>
      <location filename="change_branch.py" line="192"/>
      <source>Branch</source>
      <comment>git terminology</comment>
      <translation>Gałąź</translation>
    </message>
    <message>
      <location filename="change_branch.py" line="178"/>
      <source>Tag</source>
      <comment>git terminology</comment>
      <translation>Znacznik</translation>
    </message>
    <message numerus="yes">
      <location filename="change_branch.py" line="261"/>
      <source>Kind</source>
      <comment>Table header for git ref type (e.g. either Tag or Branch)</comment>
      <translation>
        <numerusform>Rodzaj</numerusform>
        <numerusform>Rodzaje</numerusform>
        <numerusform>Rodzaje</numerusform>
        <numerusform>Rodzaje</numerusform>
      </translation>
    </message>
    <message>
      <location filename="change_branch.py" line="265"/>
      <source>Local name</source>
      <comment>Table header for git ref name</comment>
      <translation>Nazwa lokalna</translation>
    </message>
    <message numerus="yes">
      <location filename="change_branch.py" line="271"/>
      <source>Tracking</source>
      <comment>Table header for git remote tracking branch name name</comment>
      <translation>
        <numerusform>Śledzenie</numerusform>
        <numerusform>Śledzenie</numerusform>
        <numerusform>Śledzenie</numerusform>
        <numerusform>Śledzenie</numerusform>
      </translation>
    </message>
    <message numerus="yes">
      <location filename="change_branch.py" line="277"/>
      <source>Local updated</source>
      <comment>Table header for git update time of local branch</comment>
      <translation>
        <numerusform>Lokalne zaktualizowane</numerusform>
        <numerusform>Lokalne zaktualizowane</numerusform>
        <numerusform>Lokalne zaktualizowane</numerusform>
        <numerusform>Lokalne zaktualizowane</numerusform>
      </translation>
    </message>
    <message numerus="yes">
      <location filename="change_branch.py" line="283"/>
      <source>Remote updated</source>
      <comment>Table header for git update time of remote branch</comment>
      <translation>
        <numerusform>Zdalna aktualizacja</numerusform>
        <numerusform>Zdalne aktualizacje</numerusform>
        <numerusform>Zdalne aktualizacje</numerusform>
        <numerusform>Zdalna aktualizacja</numerusform>
      </translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="191"/>
      <source>Checking connection</source>
      <translation>Testowanie połączenia</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="192"/>
      <source>Checking for connection to GitHub...</source>
      <translation>Sprawdzanie połączenia z GitHub ...</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="213"/>
      <source>Connection failed</source>
      <translation>Połączenie nie powiodło się</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="240"/>
      <source>Missing dependency</source>
      <translation>Brakująca zależność</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="324"/>
      <source>Starting up...</source>
      <translation>Rozpoczynanie ...</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="258"/>
      <source>Loading addon information</source>
      <translation>Wczytywanie informacji o dodatku</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="199"/>
      <source>Addon manager</source>
      <translation>Menedżer dodatków</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="209"/>
      <source>Restart now</source>
      <translation>Uruchom ponownie teraz</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="210"/>
      <source>Restart later</source>
      <translation>Uruchom ponownie później</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1248"/>
      <source>Refresh local cache</source>
      <translation>Odśwież pamięć podręczną</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="659"/>
      <source>Updating cache...</source>
      <translation>Aktualizowanie pamięci podręcznej ...</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="766"/>
      <source>Could not find addon &apos;{}&apos; to select
</source>
      <translation>Nie można znaleźć dodatku &apos;{}&apos; aby wybrać
</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="243"/>
      <source>Checking for updates...</source>
      <translation>Sprawdzanie aktualizacji...</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="773"/>
      <source>Apply {} update(s)</source>
      <translation>Wykonaj {} aktualizacje</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="577"/>
      <source>No updates available</source>
      <translation>Brak dostępnych aktualizacji</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="915"/>
      <source>Too many to list</source>
      <translation>Lista jest zbyt długa do wyświetlenia</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="893"/>
      <source>Missing Requirement</source>
      <translation>Brakujące wymagania</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="86"/>
      <source>The addons that can be installed here are not officially part of FreeCAD, and are not reviewed by the FreeCAD team. Make sure you know what you are installing!</source>
      <translation>Dodatki, które mogą być zainstalowane, nie są oficjalnie częścią FreeCAD, i nie są sprawdzane przez zespół FreeCAD. Upewnij się, że wiesz, co instalujesz!</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="203"/>
      <source>You must restart FreeCAD for changes to take effect.</source>
      <translation>Musisz zrestartować FreeCAD, aby zmiany zaczęły obowiązywać.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="204"/>
      <source>You must restart FreeCAD for changes to take effect. Press Ok to restart FreeCAD now, or Cancel to restart later.</source>
      <translation>Musisz zrestartować program FreeCAD, aby zmiany odniosły skutek. Naciśnij Ok, aby zrestartować FreeCAD teraz, lub Anuluj, aby zrestartować program później.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="241"/>
      <source>Could not import QtNetwork -- see Report View for details. Addon Manager unavailable.</source>
      <translation>Nie można zaimportować QtNetwork — szczegóły możesz zobaczyć w Widoku raportu. Menedżer dodatków jest niedostępny.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="262"/>
      <source>Apply</source>
      <translation>Zastosuj</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="263"/>
      <source>update(s)</source>
      <translation>aktualizacji</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="266"/>
      <source>No update available</source>
      <translation>Brak dostępnych aktualizacji</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="433"/>
      <source>Macro successfully installed. The macro is now available from the Macros dialog.</source>
      <translation>Makro zostało pomyślnie zainstalowane. Jest teraz dostępne w oknie dialogowym Makrodefinicje.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="435"/>
      <source>Unable to install</source>
      <translation>Nie można zainstalować</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="494"/>
      <source>Addon successfully removed. Please restart FreeCAD</source>
      <translation>Pomyślnie usunięto. Proszę zrestartować FreeCAD</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="496"/>
      <source>Unable to remove this addon</source>
      <translation>Nie można usunąć tego dodatku</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="502"/>
      <source>Macro successfully removed.</source>
      <translation>Makro zostało pomyślnie usunięte.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="504"/>
      <source>Macro could not be removed.</source>
      <translation>Nie można usunąć makra.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="534"/>
      <source>Addon Manager: Skipping update check because AutoCheck user preference is False</source>
      <translation>Menedżer dodatków: Pomijam sprawdzania aktualizacji, ponieważ preferencja użytkownika Sprawdź automatycznie ma wartość Fałsz</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="542"/>
      <source>Restart required</source>
      <translation>Wymagany restart</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="774"/>
      <source>All packages updates failed. Packages:</source>
      <translation>Wszystkie aktualizacje pakietów nie powiodły się. Pakiety:</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="786"/>
      <source>Failed packages:</source>
      <translation>Pakiety uszkodzone:</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1079"/>
      <source>Press OK to install anyway.</source>
      <translation>Naciśnij przycisk OK, aby pomimo to zainstalować.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="964"/>
      <source>Installing dependencies</source>
      <translation>Instalowanie zależności</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1061"/>
      <source>Cannot execute Python</source>
      <translation>Nie można wykonać skryptu Python</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1081"/>
      <source>Cannot execute pip</source>
      <translation>Nie można uruchomić pip</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1102"/>
      <source>Package installation failed</source>
      <translation>Instalacja pakietu nie powiodła się</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1105"/>
      <source>See Report View for detailed failure log.</source>
      <translation>Szczegóły zapisu awarii znajdują się w widoku raportu.</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="703"/>
      <source>Installation of macro failed</source>
      <translation>Instalacja makrodefinicji nie powiodła się</translation>
    </message>
    <message numerus="yes">
      <location filename="AddonManager.py" line="1280"/>
      <source>{} total, see Report view for list</source>
      <comment>Describes the number of updates that were completed (&apos;{}&apos; is replaced by the number of updates)</comment>
      <translation>
        <numerusform>{} ogółem, lista znajduje się w oknie Widok raportu</numerusform>
        <numerusform>{} ogółem, lista znajduje się w oknie Widok raportu</numerusform>
        <numerusform>{} ogółem, lista znajduje się w oknie Widok raportu</numerusform>
        <numerusform>{} ogółem, lista znajduje się w oknie Widok raportu</numerusform>
      </translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1178"/>
      <source>Succeeded</source>
      <translation>Zakończono z powodzeniem</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1159"/>
      <source>All packages updates failed:</source>
      <translation>Aktualizacja wszystkich pakietów nie powiodła się:</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1183"/>
      <source>Failed</source>
      <translation>Nie powiodło się</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="800"/>
      <source>Update report</source>
      <translation>Raport aktualizacji</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="850"/>
      <source>Installation succeeded</source>
      <translation>Instalacja zakończona sukcesem</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="866"/>
      <source>Installation failed</source>
      <translation>Instalacja nie powiodła się</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1457"/>
      <source>Confirm remove</source>
      <translation>Potwierdź usunięcie</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1460"/>
      <source>Are you sure you want to uninstall this Addon?</source>
      <translation>Czy na pewno chcesz odinstalować ten dodatek?</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1513"/>
      <source>Execution of Addon&apos;s uninstall.py script failed. Proceeding with uninstall...</source>
      <translation>Wykonanie skryptu dodatku uninstall.py nie powiodło się. Kontynuuję odinstalowywanie ...</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="1558"/>
      <source>Successfully uninstalled {}</source>
      <translation>Pomyślnie odinstalowano {}</translation>
    </message>
    <message>
      <location filename="addonmanager_macro.py" line="157"/>
      <source>Unable to fetch the code of this macro.</source>
      <translation>Nie można pobrać kodu makrodefinicji.</translation>
    </message>
    <message>
      <location filename="addonmanager_macro.py" line="164"/>
      <source>Unable to retrieve a description for this macro.</source>
      <translation>Nie można pobrać opisu makrodefinicji.</translation>
    </message>
  </context>
  <context>
    <name>Std_AddonMgr</name>
    <message>
      <location filename="AddonManager.py" line="68"/>
      <source>&amp;Addon manager</source>
      <translation>&amp;Menedżer dodatków</translation>
    </message>
    <message>
      <location filename="AddonManager.py" line="69"/>
      <source>Manage external workbenches and macros</source>
      <translation>Zarządzaj zewnętrznymi Środowiskami pracy i makrodefinicjami</translation>
    </message>
  </context>
  <context>
    <name>AddonInstaller</name>
    <message>
      <location filename="addonmanager_workers.py" line="535"/>
      <source>Installed location</source>
      <translation>Miejsce instalacji</translation>
    </message>
  </context>
  <context>
    <name>Dialog</name>
    <message>
      <location filename="../../AddonManager.ui" line="14"/>
      <source>Addon Manager</source>
      <translation>Menedżer dodatków</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="37"/>
      <source>Workbenches</source>
      <translation>Środowiska
    pracy</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="47"/>
      <source>Macros</source>
      <translation>Makrodefinicje</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="56"/>
      <source>Executes the selected macro, if installed</source>
      <translation>Rozpoczyna wykonywanie wybranego makra, jeśli jest zainstalowane</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="59"/>
      <source>Execute</source>
      <translation>Wykonaj</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="89"/>
      <source>Sets configuration options for the Addon Manager</source>
      <translation>Ustawia opcje konfiguracji dla Menedżera dodatków</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="92"/>
      <source>Configure...</source>
      <translation>Konfiguruj...</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="113"/>
      <source>Downloading info...</source>
      <translation>Pobieranie informacji...</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="127"/>
      <source>Uninstalls a selected macro or workbench</source>
      <translation>Odinstalowuje wybrane makrodefinicje lub środowisko pracy</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="130"/>
      <source>Uninstall selected</source>
      <translation>Odinstaluj wybrane</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="137"/>
      <source>Installs or updates the selected macro or workbench</source>
      <translation>Instaluje lub aktualizuje wybrane makrodefinicje lub środowisko pracy</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="140"/>
      <source>Install/update selected</source>
      <translation>Zainstaluj / zaktualizuj wybrane</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="147"/>
      <source>Download and apply all available updates</source>
      <translation>Pobierz i zainstaluj wszystkie dostępne aktualizacje</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="150"/>
      <source>Update all</source>
      <translation>Aktualizuj wszystko</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="157"/>
      <source>Close the Addon Manager</source>
      <translation>Zamknij Menedżera dodatków</translation>
    </message>
    <message>
      <location filename="../../AddonManager.ui" line="160"/>
      <source>Close</source>
      <translation>Zamknij</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="14"/>
      <source>Addon manager options</source>
      <translation>Opcje menedżera dodatków</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="20"/>
      <source>If this option is selected, when launching the Addon Manager,
installed addons will be checked for available updates
(this requires the GitPython package installed on your system)</source>
      <translation>Jeśli ta opcja jest zaznaczona, podczas uruchamiania Menedżera Dodatków,
zainstalowane dodatki zostaną sprawdzone pod kątem dostępnych aktualizacji
(wymaga to zainstalowanego pakietu GitPython w systemie)</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="25"/>
      <source>Automatically check for updates at start (requires GitPython)</source>
      <translation>Automatycznie sprawdzaj aktualizacje podczas uruchomienia (wymaga GitPython)</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="35"/>
      <source>Custom repositories (one per line):</source>
      <translation>Repozytoria użytkownika (jedno na linię):</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="42"/>
      <source>You can use this window to specify additional addon repositories
to be scanned for available addons</source>
      <translation>Możesz użyć tego okna, aby określić dodatkowe repozytoria dodatków,
które zostaną użyte w poszukiwaniu dostępnych dodatków</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="57"/>
      <source>Proxy </source>
      <translation>Serwer pośredniczący </translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="64"/>
      <source>No proxy</source>
      <translation>Bez serwera pośredniczącego</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="71"/>
      <source>User system proxy</source>
      <translation>Serwer pośredniczący systemu użytkownika</translation>
    </message>
    <message>
      <location filename="../../AddonManagerOptions.ui" line="78"/>
      <source>User defined proxy :</source>
      <translation>Serwer pośredniczący zdefiniowany przez użytkownika:</translation>
    </message>
  </context>
</TS>
